carbonmark-api/test/test.constants.ts

```diff
@@ -32,21 +32,39 @@ export const COUNTRIES = [
   {
     id: "A Random Country",
   },
+  {
+    id: "China",
+  },
+  {
+    id: "Spain",
+  },
+  {
+    id: "United Kingdom",
+  },
+  {
+    id: "Iceland",
+  },
+  {
+    id: "Ukraine",
+  },
+  {
+    id: "Vietnam",
+  },
 ];
 
 export const MOCK_USER_PROFILE: UserProfile = {
   address: MOCK_ADDRESS,
   createdAt: new Date("1970-01-01T00:00:00Z").getTime(),
   description: "Some description",
   handle: "SomeHandle",
-  updatedAt: new Date("2023-11-11T15:05:08Z").getTime(),
+  updatedAt: 1702675170,
   username: "someusername",
   profileImgUrl: null,
 };
 
 export const EXPECTED_USER_RESPONSE = {
   ...omit(MOCK_USER_PROFILE, ["address", "updatedAt", "createdAt"]),
-  updatedAt: 1699715108,
+  updatedAt: 1702675,
```

Comment:
Something is wrong here. I think MOCK_USER_PROFILE updated at should be something like 1702675170000